### PR TITLE
toc: optimize the document directory structure

### PR DESCRIPTION
**Pull request by scsldb**

### CODE REVIEW

File: TOC.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
-# TiDB 中文技术文档
+# TiDB 中文用户文档
 
 ## 目录
 
 + [关于 TiDB](overview.md)
+
```
Comment (lilin90) [inline TOC.md]: @YiniXu9506 Is it acceptable to leave a blank line here? @scsldb Actually, I think this blank line is unnecessary.

**Comment by queenypingcap** [comment]
LGTM.

File: TOC.md (hunk 1 above)
Comment (YiniXu9506) [reply TOC.md]: @scsldb @lilin90 Please remove the blank line here. Thanks!

File: TOC.md [hunk 2]
```diff
@@ -25,6 +26,13 @@
...
     - [使用 Ansible 部署 DM 集群](tools/dm/deployment.md)
+  + 配置
+    + [时区](sql/time-zone.md)
+    + [内存控制](sql/tidb-memory-control.md)
+    + 配置参数
+      + [PD](op-guide/pd-configuration.md)
+      + [TiDB](op-guide/configuration.md)
+      + [TiKV](op-guide/tikv-configuration.md)
   + 安全
     - [与 MySQL 的安全特性差异](sql/security-compatibility.md)
```
Comment (lilin90) [inline TOC.md]: This file is also put under "参考手册 - 安全". I think we only need to keep one?

File: TOC.md [hunk 3]
```diff
@@ -107,40 +107,40 @@
...
-      - [配置参数](op-guide/tikv-configuration.md)
+  - 安全
+      - [与 MySQL 的安全特性差异](sql/security-compatibility.md)
+      - [TiDB 数据库权限管理](sql/privilege.md)
+      - [TiDB 用户账户管理](sql/user-account-management.md)
+   + 事务
+      - [事务模型](sql/transaction-model.md)
+      - [隔离级别](sql/transaction-isolation.md)
+    - 系统表
+        - [mysql](sql/system-database.md)
```
Comment (lilin90) [inline TOC.md]: Would it be better to change `mysql` to `TiDB 系统表` (which is more accurate)?

File: TOC.md [hunk 4]
```diff
@@ -181,51 +181,22 @@
...
+- [贡献](contribute.md)
+  - [贡献代码](contribute.md#成为-tidb-的贡献者)
+  - [改进文档](contribute.md#改进文档)
+- [TiDB 路线图](ROADMAP.md)
+- [版本发布历史](releases/rn.md)
+  - v3.0
     - [3.0.0 Beta.1](releases/3.0.0-beta.1.md)
     - [3.0 Beta](releases/3.0beta.md)
-  + v2.1
+  - v2.1
```
Comment (lilin90) [inline TOC.md]: The original `+` is on purpose, to make it easier to read this TOC file. `+` indicates it has sub-levels. I'll change it back.

File: TOC.md [hunk 5]
```diff
@@ -25,6 +25,13 @@
...
+  + 配置
+    - [时区](sql/time-zone.md)
+    - [内存控制](sql/tidb-memory-control.md)
+    + 配置参数
+      - [PD](op-guide/pd-configuration.md)
+      - [TiDB](op-guide/configuration.md)
+      - [TiKV](op-guide/tikv-configuration.md)
   + 安全
     - [与 MySQL 的安全特性差异](sql/security-compatibility.md)
     - [TiDB 数据库权限管理](sql/privilege.md)
```
Comment (dcalvin) [inline TOC.md]: The same as above. Should only keep one

File: TOC.md (hunk 2 above)
Comment (scsldb) [reply TOC.md]: I recommend deleting "与 MySQL 的安全特性差异","TiDB 数据库权限管理","TiDB 用户账户管理" and keeping other

File: TOC.md (hunk 3 above)
Comment (scsldb) [reply TOC.md]: "mysql" is system database, currently, cannot be changed

Comment (lilin90) [reply TOC.md]: @scsldb But this document is all about "系统表".

File: TOC.md (hunk 2 above)
Comment (lilin90) [reply TOC.md]: @scsldb OK. Then you can update it in this PR.

File: TOC.md (hunk 3 above)
Comment (scsldb) [reply TOC.md]: The old document have some problems, there is no specific system database,eg mysql, information_schema

**Comment by morgo** [comment]
@scsldb could you please describe the changes so I can try and compare to the en `docs` TOC?

**Comment by lilin90** [comment]
The changes have been updated in #1320, so I'll close this PR.

**Pull request closed without merging**